### PR TITLE
Small bug fix on initial search

### PR DESCRIPTION
**Pull request by jesperlauridsen**
A search launched from searchbox did not use the right function, so it only triggered a normal search (and was unable to trigger an image search or URL search) This has been fixed.

URL search query magic wasn't functioning right. It has been fixed.

No JIRA on this, was just 2 mistakes i found while coding.

### CODE REVIEW

File: src/js/src/mixins/SearchUtils.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ export default {
         this.deliverImgSearchRequest(futureQuery ,updateHistory)
       }
       else if(this.solrSettings.urlSearch) {
+        console.log('we came here!')
```
Comment (jorntx) [inline src/js/src/mixins/SearchUtils.js:98]: Remember to get rid of this.

File: src/js/src/components/SearchBox.vue [hunk 2]
```diff
@@ -144,14 +144,19 @@ export default {
...
-      return this.$_validateUrlSearchPrefix(this.futureQuery) === false 
-                 ? isPrefixUrlNorm === 'url_norm' ? 'urlNotTrue' : 'urlNotTrue urlSearchActivated'
-                 : isPrefixUrlNorm === 'url_norm' ? '' : 'urlSearchActivated' 
+      return !this.$_validateUrlSearchPrefix(this.futureQuery) 
+                 ? isPrefixUrlNorm === 'url_norm' 
+                    ? 'urlNotTrue' 
+                    : 'urlNotTrue urlSearchActivated'
+                 : isPrefixUrlNorm !== 'url_norm' 
+                    ? '' 
+                    : 'urlSearchActivated'
```
Comment (jorntx) [inline src/js/src/components/SearchBox.vue:153]: Just on verge to be very hard to decipher.

**Review by jorntx — APPROVED** [review]
Good catch! 

Have a a couple of very minor comments you can look into when you find the time. Not enough to stop merging the PR though.

Approved.

**Comment by jorntx** [comment]
@jesperlauridsen can you resolve conflicts? Thanks.

**Comment by jesperlauridsen** [comment]
Fixed merge conflicts.

**Pull request merged**